react-components/src/components/RevealToolbar/SlicerButton.tsx

```diff
@@ -0,0 +1,115 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type ReactElement, useState, useEffect } from 'react';
+
+import { Box3, Plane, Vector3 } from 'three';
+
+import { useReveal } from '../RevealContainer/RevealContext';
+import { Button, Dropdown, Menu, RangeSlider } from '@cognite/cogs.js';
+
+import styled from 'styled-components';
+
+export const SlicerButton = (): ReactElement => {
+  const viewer = useReveal();
+
+  const [{ minHeight, maxHeight, topRatio, bottomRatio }, setSliceState] = useState<{
+    minHeight: number;
+    maxHeight: number;
+    topRatio: number;
+    bottomRatio: number;
+  }>({
+    minHeight: 0,
+    maxHeight: 0,
+    topRatio: 1,
+    bottomRatio: 0
+  });
```

Comment:
```suggestion
  const [sliceState, setSliceState] = useState<SlicerState>({
    minHeight: MIN_HEIGHT,
    maxHeight: MAX_HEIGHT,
    topRatio: 1,
    bottomRatio: 0
  });

  const { minHeight, maxHeight, topRatio, bottomRatio } = sliceState;
```
This would be a cleaner